sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -217,7 +217,8 @@ def test_sin_rewrite():
     assert sin(x).rewrite(cos) == cos(x - pi / 2, evaluate=False)
     assert sin(x).rewrite(sec) == 1 / sec(x - pi / 2, evaluate=False)
     assert sin(cos(x)).rewrite(Pow) == sin(cos(x))
-
+    assert (sin(x).rewrite(besselj)).equals(sqrt(2)*sqrt(pi)*sqrt(x)*besselj(S.Half, x)/2)
+    assert sin(sinh(x)).rewrite(besselj) == sqrt(2)*sqrt(pi)*sqrt(sinh(x))*besselj(S.Half, sinh(x))/2
```

Comment:
The `.equals` method should not be used here. The purpose of these tests is to test the exact form that is returned and `==` does that whereas `.equals` uses simplify and other things.